.gitignore

```diff
@@ -7,4 +7,8 @@ refresh.yml
 cover.out
 demo-deployment
 bin
-pkg/bin
\ No newline at end of file
+pkg/bin
+
+# Repos cloned for conformance testing
+internal/testing/conformance/consul-k8s/
+internal/testing/conformance/gateway-api
```

Comment:
We could, but we need a clean checkout of `main`. That's fine when running in actions in an isolated environment, but running locally could unintentionally wipe out someone's changes to those repos if they happened to be working in them. There are obviously solutions to that like stashing, but it felt more straightforward to just clone them into our own testing context. Thoughts?

Reply:
I suppose that makes sense - I was thinking it may be useful to be on different branches of both those repos when doing active development on conformance tests without making changes to the clone logic you wouldn't want to accidentally commit. Would minimize overwriting risk by only cloning if repo doesn't exist (maybe parse git branch to print a warning if it's different than expected?)

Reply:
We discussed in slack a bit ago, and the latest commits work with a checkout/directory structure:

```
GITHUB_WORKSPACE/
    consul-api-gateway/
    consul-k8s/
    gateway-api/
```